app/test_main.py

```diff
@@ -1,3 +1,24 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (50, 50, [8, 7]),
```

Comment:
The expected result `[8, 7]` for the input `(50, 50)` does not match the conversion rules provided in the task description. According to the rules, 50 cat years should convert to 8 human years and 50 dog years should convert to 7 human years. Please verify the logic in the `get_human_age` function to ensure it aligns with these rules.